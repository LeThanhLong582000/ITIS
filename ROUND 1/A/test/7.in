284
7817 11647 14632 7935 31668 217 3529 1870 6515 2009 11095 7487 30993 15596 26520 20560 14756 14881 20077 11946 23359 16270 7851 16039 17001 30960 2343 4498 4046 18065 26128 29614 8245 3439 21634 23929 18610 21146 30807 2609 8989 9694 19009 22424 32450 8098 11798 21237 18778 30224 17258 5979 1083 28022 9315 24877 6603 10220 1737 31458 10042 6200 29533 26374 3697 7150 1900 27521 25669 220 29859 25116 3825 31140 13093 29249 10435 17835 6619 6146 31509 4146 20747 7262 9599 19139 21293 25279 17739 1743 28666 12230 12585 4757 1477 19787 7460 23877 25870 12192 15061 26214 4693 22 10711 15477 23084 12483 19579 20175 27536 32130 21692 5841 27769 32314 7541 23613 15536 11876 9169 6367 23265 3527 13571 16241 8177 16006 9272 27069 17553 17501 7370 12801 25876 18728 25783 15255 30589 2725 9041 11989 26589 21882 55 11315 20188 12666 11907 31403 30357 8996 14405 30054 8615 9836 7729 31892 18568 18111 8084 28060 16669 30785 16013 26229 1160 5721 21625 22611 1858 11119 19119 11655 22377 26100 19310 1696 30043 24227 32531 21207 6787 5127 10639 18346 25203 16190 15385 31686 2611 26108 12064 10538 14809 19923 14468 23178 3162 13489 12805 2033 18848 29354 17404 26283 1757 26871 10313 28770 8478 25151 6540 25991 17955 18454 17048 20212 32622 20932 16932 17103 29594 4167 5828 28849 20974 31643 1816 14599 7218 25912 23107 21433 17089 29661 29855 18954 18144 3212 11440 18368 22350 19669 27734 1924 6941 31320 29254 24203 7871 2950 18362 11126 5988 11687 15976 11383 28804 27681 2737 182 22455 23799 19996 22771 15053 30352 9730 7020 27675 29370 4747 32513 24851 25326 18905 24982 16152 22998 19330 23993 10896 19980 