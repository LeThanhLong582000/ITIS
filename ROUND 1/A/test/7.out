20955073085950